src/store-api/src/logstore/entry.rs

```diff
@@ -32,32 +31,7 @@ pub trait Entry: Encode + Send + Sync {
     /// Return entry id that monotonically increments.
     fn id(&self) -> Id;
 
-    /// Return file offset of entry.
-    fn offset(&self) -> Offset;
-
     fn set_id(&mut self, id: Id);
```

Comment:
I searched the codebase before this patch, only found one reference to this method in tests. I think we can remove it as well. as the id is not likely to change for a wal entry